homeassistant/components/automation/time.py

```diff
@@ -34,9 +34,21 @@ def trigger(hass, config, action):
         hours, minutes, seconds = after.hour, after.minute, after.second
     elif (CONF_HOURS in config or CONF_MINUTES in config or
           CONF_SECONDS in config):
-        hours = convert(config.get(CONF_HOURS), int)
-        minutes = convert(config.get(CONF_MINUTES), int)
-        seconds = convert(config.get(CONF_SECONDS), int)
+        hours = config.get(CONF_HOURS)
+        minutes = config.get(CONF_MINUTES)
+        seconds = config.get(CONF_SECONDS)
+        if isinstance(minutes, str) and minutes.startswith('/') \
+                and not convert(minutes.lstrip('/'), int) % 60 == 0:
```

Comment:
I don't think these warnings make sense. You'll get a warning every time it is not 60, 120 etc. Better to just remove them as people might actually want to have a number not divisible by 60.